bundle/config/mutator/apply_presets.go

```diff
@@ -221,6 +222,19 @@ func (m *applyPresets) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnos
 		dashboard.DisplayName = prefix + dashboard.DisplayName
 	}
 
+	if config.IsExplicitlyEnabled((b.Config.Presets.InPlaceDeployment)) {
+		root := b.SyncRoot.Native()
+		_, ok := env.Lookup(ctx, envDatabricksRuntimeVersion)
+		isInWorkspace := ok && strings.HasPrefix(root, "/Workspace/")
```

Comment:
You can now use this function instead https://github.com/databricks/cli/pull/1889. `dbr.RunsOnRuntime()`